src/Jank/Scorepads/ui.tsx

```diff
@@ -15,19 +16,22 @@ export const Text = styled.span`
   margin: 5px 30px 5px 5px;
 `;
 
-const Button = styled.button`
+const buttonStyles = css`
```

Comment:
Since the `LoadButton` is not actually a `<button>`, but a `<Link>`, we needed to define the shared CSS like this. Note that `css` is also imported from `styled-components` - not strictly needed here, but it would allow prop interpolation if needed.